worlds/shivers/Options.py

```diff
@@ -47,13 +56,41 @@ class EarlyLightning(Toggle):
     """
     display_name = "Early Lightning"
 
+class LocationPotPieces(Choice):
+    """Chooses where pot pieces will be located within the multiworld.
+    - Own World: Pot pieces will be located within your own world
+    - Different World: Pot pieces will be located in another world
+    - Any World: Pot pieces will be located in any world"""
+    display_name = "Location of Pot Pieces"
+    option_own_world = 0
+    option_different_world = 1
+    option_any_world = 2
+
+class FullPots(Choice):
+    """Chooses if pots will be in pieces or already completed
+    - Pieces: Only pot pieces will be added to the item pool
+    - Complete: Only completed pots will be added to the item pool
+    - Mixed: Each pot will be randomly chosen to be pieces or already completed."""
```

Comment:
```suggestion
    - Mixed: Each pot will be randomly chosen to be pieces or already completed.
    """
```

Reply:
All options fixed to have """ on their own line for consistency.